doorman/static/js/plugins.js

```diff
@@ -76,4 +76,143 @@ $(function() {
+            'not_begins_with',
+            'contains',
+            'not_contains',
+            'ends_with',
+            'not_ends_with',
+            'is_empty',
+            'is_not_empty',
+            'less',
+            'less_or_equal',
+            'greater',
+            'greater_or_equal',
+        ];
+
+        var SUPPORTED_OPERATORS = SUPPORTED_OPERATOR_NAMES.map(function (operator) {
+            return QueryBuilder.OPERATORS[operator];
+        });
+
+        var COLUMN_OPERATORS = SUPPORTED_OPERATOR_NAMES.map(function (operator) {
+            return {
+                type: 'column_' + operator,
+                nb_inputs: QueryBuilder.OPERATORS[operator].nb_inputs + 1,
+                multiple: true,
+                apply_to: ['string'],        // Currently, all column operators are strings
+            };
+        });
+
+        var SUPPORTED_COLUMN_OPERATORS = SUPPORTED_OPERATOR_NAMES.map(function (operator) {
+            return 'column_' + operator;
+        });
+
+        // Copy existing names
+        var CUSTOM_LANG = {};
+        SUPPORTED_OPERATOR_NAMES.forEach(function (op) {
+            CUSTOM_LANG['column_' + op] = QueryBuilder.regional.en.operators[op];
+        });
+
+        // Get existing rules, if any.
+        var existingRules;
+        try {
+            var v = $('#rules-hidden').val();
+            if (v) {
+                existingRules = JSON.parse(v);
+            }
+        } catch (e) {
+            // Do nothing.
+        }
+
+        $queryBuilder.queryBuilder({
+            filters: [
+                {
+                    id: 'query_name',
+                    type: 'string',
+                    label: 'Query Name',
+                    operators: SUPPORTED_OPERATOR_NAMES,
+                },
+                {
+                    id: 'action',
+                    type: 'string',
+                    label: 'Action',
+                    operators: SUPPORTED_OPERATOR_NAMES,
+                },
+                {
+                    id: 'host_identifier',
+                    type: 'string',
+                    label: 'Host Identifier',
+                    operators: SUPPORTED_OPERATOR_NAMES,
+                },
+                {
+                    id: 'timestamp',
+                    type: 'integer',
+                    label: 'Timestamp',
+                    operators: SUPPORTED_OPERATOR_NAMES,
+                },
+                {
+                    id: 'column',
+                    type: 'string',
+                    label: 'Column',
+                    operators: SUPPORTED_COLUMN_OPERATORS,
+                    placeholder: 'value',
+                },
+            ],
+
+            operators: SUPPORTED_OPERATORS.concat(COLUMN_OPERATORS),
+
+            lang: {
+                operators: CUSTOM_LANG,
+            },
+
+            plugins: {
+                'bt-tooltip-errors': {
+                    delay: 100,
+                    placement: 'bottom',
```

Comment:
Add the sortable plugin, to make rules easily modifiable.

![image](https://cloud.githubusercontent.com/assets/359029/15580782/e17d29d0-2337-11e6-8c3d-fbbd7365e775.png)


```
+++ b/doorman/static/js/plugins.js
@@ -180,6 +180,9 @@ $(function() {
                     delay: 100,
                     placement: 'bottom',
                 },
+                'sortable': {
+                    icon: 'glyphicon glyphicon-move'
+                },
```